rules/test.bzl

```diff
@@ -58,9 +59,12 @@ def _ios_test(name, test_rule, test_suite_rule, apple_library, infoplists_by_bui
 
     library = apple_library(name = name, namespace_is_module_name = False, platforms = {"ios": ios_test_kwargs.get("minimum_os_version")}, **kwargs)
 
+    # TODO: make this conditional
+    import_middleman(name = name + ".import_middleman", deps = library.lib_names)
     rule(
         name = name,
-        deps = library.lib_names,
+        # deps = library.lib_names,
+        deps = [name + ".import_middleman"],
```

Comment:
I assume we want this to be configurable by a config_setting instead of replacing the current implementation right?